src/mgr/ActivePyModule.cc

```diff
@@ -122,9 +123,12 @@ int ActivePyModule::handle_command(
   PyFormatter f;
   cmdmap_dump(cmdmap, &f);
   PyObject *py_cmd = f.get();
+  string instr;
```

Comment:
const string&

Reply:
I may continue to be confused, but, I have to modify this string to put the contents of the bufferlist into it...how can it be const?

Reply:
yeah totally, i misread this as an argument list